lib/Target/Cpp/TranslationFlags.cpp

```diff
@@ -0,0 +1,31 @@
+//===- TranslationFlags.cpp - EmitC translation flags -----------*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "emitc/Target/TranslationFlags.h"
+#include "emitc/Target/Cpp.h"
+
+#include "llvm/Support/CommandLine.h"
+
+namespace mlir {
+namespace emitc {
+
+static llvm::cl::opt<bool> forwardDeclareFlag{
+    "forward-declare-variables",
+    llvm::cl::desc(
+        "Forward declares all variables at the top of emitted functions."),
```

Comment:
```suggestion
        "Forward declare all variables emitted from emitc"),
```

"at the top .. functions" might be misleading?